### PR TITLE
feat: add variables colors

### PR DESCRIPTION
**Pull request by oSlavivna**
adding colors from Figma's page UI kit

### CODE REVIEW

File: src/theme/_variables.scss [hunk 1]
```diff
@@ -1,15 +1,48 @@
 // Colors
+$white-text: #fcffff;
+$active-buttons: #f7a400;
+$hover-buttons: #fb7812;
+
+$background-one: #fff5ee;
+$background-two: #a6b7a9;
+$background-reserved: #457b6d;
+$header: #114a3e;
```
Comment (biryukov-m) [inline src/theme/_variables.scss:9]: Header should be 80% transparent

**Review by biryukov-m — CHANGES_REQUESTED** [review]
Change header color to 80% transparent

File: src/theme/_variables.scss (hunk 1 above)
Comment (oSlavivna) [reply src/theme/_variables.scss]: > Header should be 80% transparent

ok. I used rgb colors 
rgb(17, 74, 62, 80)

**Review by kholodihor — DISMISSED** [review]

File: src/theme/_variables.scss [hunk 2]
```diff
@@ -6,7 +6,7 @@ $hover-buttons: #fb7812;
 $background-one: #fff5ee;
 $background-two: #a6b7a9;
 $background-reserved: #457b6d;
-$header: #114a3e;
+$header: rgb(17, 74, 62, 80);
```
Comment (biryukov-m) [inline src/theme/_variables.scss:9]: Use correct syntax
Screenshot 2024-09-18 at 16 24 08
https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/rgb

**Review by biryukov-m — CHANGES_REQUESTED** [review]
Use correct syntax
https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/rgb

**Review by biryukov-m — APPROVED** [review]

**Pull request merged**